meta-xt-control-domain/recipes-guest/domu/files/domu-create-ExecStart.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+echo "Sleeping 5 seconds" && sleep 5;
+
+echo "Creating DomU" && /usr/sbin/xl -v create /etc/xen/domu.cfg && \
+echo "Created DomU" && \
+echo "Pausing DomU" && /usr/sbin/xl -v pause DomU && \
+echo "Paused DomU" && \
+DOMD_ID=$(xl list | awk '{ if ($1 == "DomD") print $2 }') && \
+DOMU_ID=$(xl list | awk '{ if ($1 == "DomU") print $2 }') && \
+echo "Parsed DOMD_ID is '${DOMD_ID}'" && \
+echo "Parsed DOMU_ID is '${DOMU_ID}'" && \
+echo "Writing QEMU command to xen-store" && \
+/usr/bin/xenstore-write /local/domain/${DOMD_ID}/drivers/dom0-qemu-command-monitor/value \
+"echo \"QEMU starting.\"; \
+export SDL_VIDEODRIVER=wayland; \
+qemu-system-aarch64 \
+-xen-domid ${DOMU_ID} \
+-xen-attach \
+-M xenpv \
+-m $(grep "memory =" < /etc/xen/domu.cfg | sed -rne 's/^.*memory = ([0-9]+).*/\1/p') \
+-smp $(grep "vcpus =" < /etc/xen/domu.cfg | sed -rne 's/^.*vcpus = ([0-9]+).*/\1/p') \
+-d guest_errors \
+-monitor telnet:127.0.0.1:1234,server,nowait \
+-device virtio-net-pci,disable-legacy=on,iommu_platform=on,romfile=\"\",id=nic0,netdev=net0,mac=08:00:27:ff:cb:cf \
+-netdev type=tap,id=net0,ifname=vif-emu,br=xenbr0,script=no,downscript=no \
+-device virtio-blk-pci,scsi=off,disable-legacy=on,iommu_platform=on,drive=image \
+-drive if=none,id=image,format=raw,file=/dev/mmcblk0p3 \
```

Comment:
In many cases we boot DomU via NFS. How this is handled in your case?

Reply:
Hi @lorc, accepted and done.

I've changed the implementation to support DomU boot via the NFS. It works.

The change required additional commit to the meta-xt-prod-devel-rcar:
https://github.com/xen-troops/meta-xt-prod-devel-rcar/pull/101/commits/ddad8aab2a9942d5932dfee7a32bb8daec756fb8

Done.